package.json

```diff
@@ -79,5 +80,8 @@
     "tsconfig-paths": "4.2.0",
     "tus-js-client": "^4.2.3",
     "typescript": "5.6.3"
+  },
+  "dependencies": {
```

Comment:
All has to be dev deps never dependency because we bundle them in the binary.
We already use `@std/semver` so better to see option with it